api/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
-
+from django.utils import timezone
 # Create your models here.
+
+class Task (models.Model):
+
+	STATUS_CHOICES = (
+		('UNASSIGNED', 'Unassigned'),
+		('WORKINGON', 'Working On'),
+		('COMPLETED', 'Completed'),
+		)
+
+	title = models.CharField(max_length=255, default=None)
```

Comment:
Why did you use default=None? I believe a task should have at least a Title so this would be a mandatory field.